core/domain/topic_jobs_one_off_test.py

```diff
@@ -199,3 +199,98 @@ def test_migration_job_fails_with_invalid_topic(self):
                      [u'Topic topic_id failed validation: '
                       'Invalid language code: invalid_language_code']]]
         self.assertEqual(expected, [ast.literal_eval(x) for x in output])
+
+
+class PopulateTopicThumbnailSizeOneOffJobTests(test_utils.GenericTestBase):
```

Comment:
Where's the "happy path" test (i.e. job updates the size normally)?

Reply:
Updated the test for "happy path".